### ISSUE TITLE
Clean up config loading (and remove window.ENV dependency)

### ISSUE THREAD

**Issue by tasn** [zaubererty/ember-cli-django-rest-auth#8]
It shouldn't be using the same namespace as ember-simple-auth, but its own.
Also, it shouldn't depend on windows.ENV, but instead just load its own config.
See https://github.com/simplabs/ember-simple-auth/tree/master/packages/ember-simple-auth-devise/lib/simple-auth-devise
For how it should be done.

**Comment by zaubererty** [comment]
Is this completed by your merge request #9 ?

**Comment by tasn** [comment]
No. It shouldn't be too hard to implement, I just never gotten around to it...

**Comment by tasn** [comment]
Also, things have kinda changed in ESA 1.0, so "best practices" for authenticators have probably changed a bit too. Need to look into this.

**Comment by esistgut** [comment]
any news about this?
